### ISSUE TITLE
Missing 3D Models

### ISSUE THREAD

**Issue by igorbajena** [thedworak/dfg_3dviewer#42]
After our breakdown in 3D Repository, I've started to reupload models. While it was time consuming task, it was passed to one of the student assistants in Jena. However, until today models are missing. List of status of all the entries is available here:

https://docs.google.com/spreadsheets/d/1eS23ZHGmSIisFMrCKpzdpXN8g7usI906/edit?usp=sharing&ouid=100600621836022880260&rtpof=true&sd=true

All positions, which have red cross in the column "Link" are missing.

Is someone from Jena still working on it?

**Comment by ClemensBeck** [comment]
Yes, it was Liv's Task, but there were some problems with her working contract, so it was stalled. Since she rejoined our team yesterday, I will talk to her

**Comment by ClemensBeck** [comment]
Liv and I added all missing models. Issue can be closed
